services/language_translation/v1.js

```diff
@@ -13,55 +13,208 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  **/
-
-module.exports = function(RED) {
+module.exports = function (RED) {
   var watson = require('watson-developer-cloud');
   var cfenv = require('cfenv');
+  var fs = require('fs');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['fileType' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2398740744/issues/source?bid=3225115&fileBranchId=3231190#l20)